style.css

```diff
@@ -532,3 +531,206 @@ input[type="submit"]:disabled {
 form#f2 {
   display: none;
 }
+
+/* start of desktop version design */
+@media only screen and (min-width: 768px) {
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-around;
+    width: 100%;
+    font-size: 20px;
+    height: 72px;
+    padding-top: 10px;
+  }
+
+  nav > img {
+    display: none;
+  }
+
+  nav ul {
+    display: flex;
+    justify-content: center;
+    width: 50%;
+    padding-top: 7px;
+  }
+
+  nav ul li {
+    list-style-type: none;
+    margin: 10px;
+  }
+
+  nav ul li a {
+    color: #fff;
+    font-size: 15px;
+    text-decoration: none;
+  }
+
+  #headline {
+    width: 100%;
+    height: 100vh;
+    background-image: url("Resources/Images/Illustration-header.png");
+    background-size: 47%;
+    background-position: 100%;
+    background-position-y: 10px;
+  }
+
+  #headline > ul {
+    position: relative;
+    float: left;
+    top: -21rem;
+    gap: 0.8rem;
+    flex-direction: column;
+  }
+
+  #headline > #mainhead {
+    font-size: 55px;
+    margin-top: 20px;
+    width: 80%;
+    margin-left: 80px;
+    line-height: 4rem;
+  }
+
+  #headline > .subhead {
+    font-size: 25px;
+    margin-top: 5px;
+    width: 80%;
+    margin-left: 80px;
+  }
+
+  #headline > p {
+    font-size: 16px;
+    width: 55%;
+    margin-top: 5px;
+    margin-left: 80px;
+    line-height: 2rem;
+  }
+
+  .headtext {
+    display: flex;
+    width: 100%;
+    height: 250px;
+    padding-left: 30px 20px;
+  }
+
+  #work > .first-box > .headtext > h2 {
+    border-bottom: none;
+    font-family: 'Inter', sans-serif;
+    font-size: 30px;
+    font-weight: 500;
+    text-align: center;
+    position: relative;
+    max-width: 600px;
+    margin-left: 10px;
+  }
+
+  /* to position hr */
+  .first-box > .ohh {
+    display: block;
+    position: relative;
```

Comment:
- [ ] Kindly make sure to follow the Figma design and make all the changes according to the loom video attached.
Like the spacing, font family, font size, hover on the cards etc.

Reply:
noted thanks